WRO-6720: Fix stats filter to show snapshot_blob.bin log (#292)
enactjs/cli: CHANGELOG.md

## unreleased

### pack

> * Fixed a `printFileSizes` issue that print build files without the snapshot binary.

Reviewer: ```suggestion
* Fixed `snapshot_blob.bin` file is not listed on the build results.
```
